src/System.Management.Automation/FormatAndOutput/DefaultFormatters/DotNetTypes_format_ps1xml.cs

```diff
@@ -69,6 +69,10 @@ internal static IEnumerable<ExtendedTypeDefinition> GetFormatData()
                 "System.Diagnostics.Process#IncludeUserName",
                 ViewsOf_System_Diagnostics_Process_IncludeUserName());
 
+            yield return new ExtendedTypeDefinition(
+                "System.Diagnostics.Process#IncludeCommandLine",
```

Comment:
Seems like we only have views for UserName or CommandLine and not both?  Since UserName isn't long, perhaps it's ok to have a single view with both for extended information?

Reply:
Done.